src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/ApprenticeshipEmailAddressChangedByApprentice/ApprenticeshipEmailAddressChangedByApprenticeCommandHandler.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.CommitmentsV2.Application.Commands.ApprenticeshipEmailAddressConfirmed;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Domain.Exceptions;
+using SFA.DAS.CommitmentsV2.Domain.Interfaces;
+using SFA.DAS.CommitmentsV2.Models.Api;
+using SFA.DAS.CommitmentsV2.Models.Api.Types;
+using SFA.DAS.CommitmentsV2.Shared.Extensions;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Application.Commands.ApprenticeshipEmailAddressChangedByApprentice
+{
+    public class ApprenticeshipEmailAddressChangedByApprenticeCommandHandler : AsyncRequestHandler<ApprenticeshipEmailAddressChangedByApprenticeCommand>
+    {
+        private readonly Lazy<ProviderCommitmentsDbContext> _db;
+        private readonly IApiClient _apimClient;
```

Comment:
just a personal perference, instead of calling the variable apimClient, can we call it something like emailApimClient. But leave it to you if you want to make this change.